Text check

SmausJakub opened the pull request:
From 0 to 4. Mission

SmausJakub committed:
1. intro typo

SmausJakub committed:
Some typos and maybe enhancements, depends on the creator, what we thinks

Kossyr reviewed (changes requested):
Please finish up to 11 before I review

SmausJakub committed:
Really small changes, not sure about AT home
Confident US General wouldn't use yet  in possitive sentence about killing enemy, and Gaston should know that. Please, consider, otherwise justify it with reason and reason only. It is about the story, not about the author. If you want to use yet, this is not the place.

SmausJakub committed:
Clarified, repaired.
